bitnami/suitecrm/templates/NOTES.txt

```diff
@@ -1,104 +1,43 @@
-{{- if or .Values.mariadb.enabled .Values.externalDatabase.host -}}
-
-{{- if empty (include "host" .) -}}
-###############################################################################
-### ERROR: You did not provide an external host in your 'helm install' call ###
-###############################################################################
+{{- $suitecrmSecretName := include "suitecrm.fullname" . -}}
 
-This deployment will be incomplete until you configure SuiteCRM with a resolvable
-host. To configure SuiteCRM with the URL of your service:
```

Comment:
No, I have changed the logic to use `127.0.0.1` by default when using `ClusterIP` so suitecrm will run and you can port-forward the svc or the pod.
When using NodePort or LoadBalancer it will show the warning and how to update the host with the proper IP.

Reply:
Why did you change that? It's a web app, and we're exposing externally web apps by default.

Reply:
Changed back to LoadBalancer by default